### PR TITLE
JDK-8253936 Replace <code>...</code> with {@code ...} for java.sql

### PR DESCRIPTION
**Pull request by Vipin-Sharma**
<code>...</code> is replaced with {@code ...} in java.sql classes.
Please review and sponsor this change.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Testing

|     | Linux x64 | Windows x64 | macOS x64 |
| --- | ----- | ----- | ----- |
| Build | ✔️ (5/5 passed) | ✔️ (2/2 passed) | ✔️ (2/2 passed) |
| Test (tier1) | ✔️ (9/9 passed) | ✔️ (9/9 passed) | ✔️ (9/9 passed) |

### Issue
 * [JDK-8253936](https://bugs.openjdk.java.net/browse/JDK-8253936): Replace <code>...</code> with {@code ...} for java.sql

### Reviewers
 * [Lance Andersen](https://openjdk.java.net/census#lancea) (@LanceAndersen - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/707/head:pull/707`
`$ git checkout pull/707`

### CODE REVIEW

**Comment by Vipin-Sharma** [comment]
Hi @LanceAndersen,

I have generated javadoc before and after change. Following are 3 differences showing side by side comparison here.

DriverManager.java - line 417 (Change: CODE replaced with code)
<CODE>d.getClass().getName()</CODE></div>		      |	 <code>d.getClass().getName()</code></div>

ResultSet.java - line 3085 (Change: CODE replaced with code)
the result set concurrency is <CODE>CONCUR_READ_ONLY</code>  |	 the result set concurrency is <code>CONCUR_READ_ONLY</code>

SQLClientInfoException.java - line 60 (Change: removed space before null)
 <code> null</code> and the vendor code is initialized to 0.  |	 <code>null</code> and the vendor code is initialized to 0.
 
All the above three changes doesn't change the effective view.

File: src/java.sql/share/classes/java/sql/RowId.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2006, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2006, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/RowId.java:2]: Please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLClientInfoException.java [hunk 2]
```diff
@@ -28,18 +28,18 @@
...
+ * could not be set on a {@code Connection}.  In addition to the information provided
+ * by {@code SQLException}, a {@code SQLClientInfoException} provides a list of client info
  * properties that were not set.
  *
  * Some databases do not allow multiple client info properties to be set
  * atomically.  For those databases, it is possible that some of the client
- * info properties had been set even though the <code>Connection.setClientInfo</code>
- * method threw an exception.  An application can use the <code>getFailedProperties </code>
+ * info properties had been set even though the {@code Connection.setClientInfo}
+ * method threw an exception.  An application can use the {@code getFailedProperties }
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLClientInfoException.java:38]: Please remove the extra space after getFailedProperties

File: src/java.sql/share/classes/java/sql/SQLDataException.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2010, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2010, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLDataException.java:2]: Please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLException.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1996, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1996, 2014, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLFeatureNotSupportedException.java [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLFeatureNotSupportedException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLIntegrityConstraintViolationException.java [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLIntegrityConstraintViolationException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLInvalidAuthorizationSpecException.java [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLInvalidAuthorizationSpecException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLNonTransientConnectionException.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLNonTransientConnectionException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLNonTransientConnectionException.java [hunk 9]
```diff
@@ -122,14 +122,14 @@ public SQLNonTransientConnectionException(Throwable cause) {
     }
 
     /**
-     * Constructs a <code>SQLTransientException</code> object
+     * Constructs a {@code SQLTransientException} object
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLNonTransientConnectionException.java:125]: This should be SQLNonTransientConnectionException

File: src/java.sql/share/classes/java/sql/SQLNonTransientException.java [hunk 10]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLNonTransientException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLNonTransientException.java [hunk 11]
```diff
@@ -119,14 +119,14 @@ public SQLNonTransientException(Throwable cause) {
     }
 
     /**
-     * Constructs a <code>SQLTransientException</code> object
+     * Constructs a {@code SQLTransientException} object
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLNonTransientException.java:122]: Should be SQLNonTransiientException

File: src/java.sql/share/classes/java/sql/SQLPermission.java [hunk 12]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1999, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1999, 2017, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLPermission.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLRecoverableException.java [hunk 13]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2006, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2006, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLRecoverableException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLSyntaxErrorException.java [hunk 14]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLSyntaxErrorException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLTimeoutException.java [hunk 15]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1996, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1996, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLTimeoutException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLTransactionRollbackException.java [hunk 16]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLTransactionRollbackException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLTransientConnectionException.java [hunk 17]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLTransientConnectionException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLTransientException.java [hunk 18]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLTransientException.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/SQLWarning.java [hunk 19]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1996, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1996, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/SQLWarning.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/Types.java [hunk 20]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1996, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1996, 2014, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/Types.java:2]: please fix the copyright

File: src/java.sql/share/classes/java/sql/Wrapper.java [hunk 21]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2013, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/java/sql/Wrapper.java:2]: please fix the copyright

File: src/java.sql/share/classes/javax/sql/ConnectionEvent.java [hunk 22]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2006, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2006, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/javax/sql/ConnectionEvent.java:2]: please fix the copyright

File: src/java.sql/share/classes/javax/sql/ConnectionEventListener.java [hunk 23]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2001, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2001, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/javax/sql/ConnectionEventListener.java:2]: please fix the copyright

File: src/java.sql/share/classes/javax/sql/RowSetEvent.java [hunk 24]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2006, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2006, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/javax/sql/RowSetEvent.java:2]: please fix the copyright

File: src/java.sql/share/classes/javax/sql/RowSetListener.java [hunk 25]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, 2001, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2001, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/javax/sql/RowSetListener.java:2]: please fix the copyright

File: src/java.sql/share/classes/javax/sql/StatementEventListener.java [hunk 26]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2014, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (LanceAndersen) [inline src/java.sql/share/classes/javax/sql/StatementEventListener.java:2]: please fix the copyright

**Review by LanceAndersen — CHANGES_REQUESTED** [review]
Hi Vipin

Thank you for tackling this update.  There are a few required changes below.  I will also be going through the generated javadoc to see if anything jumps out as well.

Best
Lance

**Review by LanceAndersen — APPROVED** [review]

**Comment by Vipin-Sharma** [comment]
I think all steps are completed for this PR, please help sponsoring this.

**Pull request closed without merging**